.github/workflows/deploy.yml

```diff
@@ -0,0 +1,62 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - main
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && ${{ github.ref }} == 'main'
```

Comment:
do we need this now?

Reply:
No we don't need line 10 now, I will remove and push